content/en-us/projects/performance-optimization/computation.md

```diff
@@ -102,9 +102,10 @@ computation to be expensive.
   of `Class.RunService` without limiting the frequency, means these operations
   will be repeated every frame which often results in an unnecessary increase in
   computation time. These events include:
-  - `Class.RunService.RenderStepped`
-  - `Class.RunService.Stepped`
-  - `Class.RunService.Heartbeat`
```

Comment:
Keep `Heartbeat`

Reply:
Did you want me to replace Class.RunService.PostSimulation with Class.RunService.Heartbeat or add it to the list?